applications/ParticleMechanicsApplication/python_scripts/particle_mpm_solver.py

```diff
@@ -29,6 +59,11 @@ def __init__(self, grid_model_part, initial_material_model_part, material_model_
         # Default settings string in json format
         default_settings = KratosMultiphysics.Parameters("""
         {
+            "model_part_name" : "MPM_Material",
+            "domain_size"     : 2,
```

Comment:
please put some unreasonable value here (e.g. -1) and check afterwards if the user assigned sth
=> see the same in FluidDynamicsApp and StructuralMechanicsApp